Automattic/wp-calypso
# Replace coded preview with an image

Related to SELFSERVE-539

## Proposed Changes

* Replace previously coded banner preview with an image as Bart suggested in p1685470242826819-slack-C053FAERCDP.

## Testing Instructions



* Use can use live preview or run Calypso locally
* Open Tools > Advertising section
* Ensure your test user has less than 3 campaigns, otherwise a banner disappears
* Try various screen widths:
  - Less than `1280px`, no banner should be shown
  - From `1280px` to `1439px` (inclusive), the banner should be located in the bottom right corner
  - From `1440px` and wider, the banner should satisfy main column's width

Screencast of testing:

https://github.com/Automattic/wp-calypso/assets/115007291/5f898944-2c91-4eaf-a431-c9ed67bb78b9

## Pre-merge Checklist
